i18n/update_translations.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
```

Comment:
I think we should merge this for now, and think about a way, to integrate the functionality of the python script better in our build chain, later on.